config/default.yml

```diff
@@ -737,12 +737,19 @@ PreCommit:
     install_command: 'brew install swiftlint'
     include: '**/*.swift'
 
+  Terraform:
```

Comment:
I would say we should call this `TerraformFormat`, as that is ultimately what it is checking. While I know Terraform is a pretty ubiquitous tool at this point, I think for some users just seeing "Terraform" doesn't necessarily hint at what it is doing.